<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fcf0ca;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_well">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: #bde0ff;
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
    </property>
    <property name="text">
     <string>Колодец</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_lizard">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: #bde0ff;
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
    </property>
    <property name="text">
     <string>Ящерица</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_spock">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: #bde0ff;
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
    </property>
    <property name="text">
     <string>Спок</string>
    </property>
   </widget>
   <widget class="Line" name="design_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: #bde0ff;
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
    </property>
    <property name="text">
     <string>Камень</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_scissors">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: #bde0ff;
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
    </property>
    <property name="text">
     <string>Ножницы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_papers">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: #bde0ff;
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
    </property>
    <property name="text">
     <string>Бумага</string>
    </property>
   </widget>
   <widget class="QLabel" name="design_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>420</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border-style: outset;
border-color: rgb(0, 0, 0);
border-width: 1px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="design_label2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Пользователь</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="user_score">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="computer_score">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="design_label3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Компьютер</string>
    </property>
   </widget>
   <widget class="QLabel" name="design_label2_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>- Cчёт игры -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="design_label5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Ваш ход:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_change">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="design_label4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Мой ход:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="computer_change">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="design_label4_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Раунд </string>
    </property>
   </widget>
   <widget class="QLabel" name="design_label_tablo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
